src/components/networked-counter.js

```diff
@@ -5,77 +5,54 @@
  */
 AFRAME.registerComponent("networked-counter", {
   schema: {
-    max: { default: 3 },
-    ttl: { default: 0 },
```

Comment:
I removed `ttl` because we weren't using it and that made it easier to rework, but I can bring it back if it's something we care about.